usaspending_api/search/migrations/0041_alter_awardsearch_award_alter_summarystateview_duh.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 5.0.4 on 2024-04-15 19:24
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Hey Ayub, we purposely avoided creating a FK here in the database. We simulated the relationship in Django here: 
https://github.com/fedspendingtransparency/usaspending-api/blob/qat/usaspending_api/search/migrations/0008_awardsearch_table.py#L116

Reply:
After looking more, I can see that this probably has to do with adding `serlialize=False` to primary keys. Not sure if we need to update this migration to include the change to Award Search in a state migration like in the linked migration

Reply:
After researching a bit, I think we should be in good shape here. The change should only apply to the Django state, not the database schema. 

Can you please double check that this is the case in your environment after applying the migration? (No FK exists from `award_search` to `awards`)